tests/TestsNextGen/Unit/DonationForm/Models/TestDonationForm.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace TestsNextGen\Unit\DonationForm\Models;
+
+use Exception;
+use Give\Donations\Models\Donation;
+use Give\Donations\Models\DonationNote;
+use Give\Donations\ValueObjects\DonationStatus;
+use Give\Donors\Models\Donor;
+use Give\Framework\Support\ValueObjects\Money;
+use Give\NextGen\DonationForm\Models\DonationForm;
+use Give\PaymentGateways\Gateways\TestGateway\TestGateway;
+use Give\Subscriptions\Models\Subscription;
+use GiveTests\TestCase;
+use GiveTests\TestTraits\RefreshDatabase;
+
+/**
+ * @unreleased
+ */
+class TestDonationForm extends TestCase
```

Comment:
Let's make sure to add a suite of tests for the DonationFormRepository as well.  All the other models have both model & repository tests. There's a little bit of overlap between the model and repository in terms of testing but there's definitely some nuances that make it worth it.

Reply:
Resolved by https://github.com/impress-org/givewp-next-gen/pull/63/commits/5122a3ce7bfe4044bab8c5b7ac1c3cd914df7340.